py-polars/src/lib.rs

```diff
@@ -225,13 +225,31 @@ fn polars(py: Python, m: &PyModule) -> PyResult<()> {
         functions::string_cache::using_string_cache
     ))
     .unwrap();
-    m.add_wrapped(wrap_pyfunction!(functions::meta::set_float_fmt))
+
+    // Numeric formatting
+    m.add_wrapped(wrap_pyfunction!(functions::meta::get_digit_group_separator))
+        .unwrap();
```

Comment:
Just for my understanding: why are some of  the Config options implemented as functions in the Rust bindings, while others go through environment variables?

Reply:
Latency, I believe; want to avoid repeated env var lookup in a loop.